src/main/java/org/sigmah/shared/dto/referential/ProjectModelStatus.java

```diff
@@ -134,6 +134,9 @@ public static ProjectModelStatus getStatus(final String status) {
 			// "USED" status is only allowed to shift to "UNAVAILABLE" status.
 			return new Pair<Boolean, String>(false, I18N.CONSTANTS.usedModelStatusChangeError());
 
+		} else if (currentStatus == ProjectModelStatus.USED && !currentModel.isUnderMaintenance() && targetStatus == ProjectModelStatus.DRAFT) {

```

Comment:
This test doesn't seem to match with issue description. Your line of code sends an error if a project model status which is Used and NOT under maintenance is changed to Draft. But actually, the issue appears for project model in Used and under maintenance status.